source/get-started/create-a-connection-string.txt

```diff
@@ -0,0 +1,65 @@
+.. _scala-connection-string:
+
+==========================
+Create a Connection String
+==========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: uri, atlas
+
+You can connect to your MongoDB deployment by providing a
+**connection URI**, also called a *connection string*, which
+instructs the driver on how to connect to a MongoDB deployment
+and how to behave while connected.
+
+The connection string includes the hostname or IP address and 
+port of your deployment, the authentication mechanism, user credentials 
+when applicable, and connection options.
+
+.. TODO:
+ To connect to an instance or deployment not hosted on Atlas, see
+ :ref:`scala-connection-targets`.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Find your MongoDB Atlas Connection String
+
+      To retrieve your connection string for the deployment that
+      you created in the :ref:`previous step <scala-create-deployment>`,
+      log in to your Atlas account and navigate to the
+      :guilabel:`Database` section and click the :guilabel:`Connect` button
+      for your new deployment.
+
+      .. figure:: /includes/figures/atlas_connection_select_cluster.png
+         :alt: The connect button in the clusters section of the Atlas UI
+
+      Then, select the :guilabel:`Drivers` option under the :guilabel:`Connect to your application`
+      header. Select "Scala" from the :guilabel:`Driver` selection
+      menu and the version that best matches the version you installed
+      from the :guilabel:`Version` selection menu.
+
+   .. step:: Copy your Connection String
+
+      Click the button on the right of the connection string to copy it
+      to your clipboard, as shown in the following screenshot:
+
+      .. figure:: /includes/figures/atlas_connection_copy_string_scala.png
+         :alt: The copy button next to the connection string in the Atlas UI
+
+   .. step:: Update the Placeholders
+
+      Paste this connection string into a file in your preferred text editor
+      and replace the ``<db_username>`` and ``<db_password>`` placeholders with
```

Comment:
Given the ticket we just got to revert to just using `username` and `password`, I think we can lose the `db_` here.

Reply:
The Atlas UI is showing `db_username` and `db_password` as the placeholders - I think it makes sense to leave here, since it's Atlas-related and describing what's in the screenshot